packages/browser-sdk/src/lib/embed/index.ts

```diff
@@ -82,8 +82,8 @@ interface WherebyEmbedElementEventMap {
     participantupdate: CustomEvent<{ count: number }>;
     join: CustomEvent;
     leave: CustomEvent<{ removed: boolean }>;
-    participant_join: CustomEvent<{ participant: { metadata: string } }>;
-    participant_leave: CustomEvent<{ participant: { metadata: string } }>;
+    participant_join: CustomEvent<{ participant: { metadata: string, externalId: string } }>;
```

Comment:
Both of these values can actually be empty (`null`). Therefore this type definition should be updated to the following:

```suggestion
    participant_join: CustomEvent<{ participant: { metadata: string | null, externalId: string | null } }>;
    participant_leave: CustomEvent<{ participant: { metadata: string | null, externalId: string | null } }>;
```